.github/workflows/handle-closed-prs.yml

```diff
@@ -51,7 +51,7 @@ jobs:
           aws-region: 'eu-west-1'
           aws-access-key-id: ${{ secrets.AWS_KEY_ID }}
           aws-secret-access-key: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
-          environment-name: nextjs-app-pr-${{ github.event.number }}
+          environment-name: nextjs-app-v13-pr-${{ github.event.number }}
```

Comment:
Is there a corresponding place where the environment with this name is spun up? I can't seem to see it

Reply:
Yeah, this is just the corresponding name for the deployment which already incorporates the new `v13` name change.